### DISCUSSION TITLE
CTFFIND5

### DISCUSSION THREAD

**Discussion by xingzhe1129** [nextpyp/.github#110] [Ideas]
Is it possible to add CTFFINFD5 into nextPYP? CTFFIND5 is from cisTEM ( https://cistem.org/development ). And the paper about CTFFIND5 is here ( https://www.biorxiv.org/content/10.1101/2024.02.26.582023v1 ). Could you please try to pull CTFFIND5 from cistem and add into nextPYP? Thanks.

**Comment by abartesaghi** [comment]
Yes, this is on our roadmap. Thanks for the suggestion!
